website/pages/docs/reference/source-spec.md

```diff
@@ -142,3 +126,28 @@ If set to `true`, the exporter will not verify the server will connect via `http
 (`object`, optional)
 
 Plugin-specific configurations. Visit [source plugins](/docs/plugins/sources/overview) documentation for more information.
+The following options are available for most source plugins:
```

Comment:
Oh I missed this line. I think we should remove it from this page and move it to each plugin page where applicable (but understand if you don't want to do it in this PR necessarily)

Reply:
Maybe this line should be under a new heading at least

Reply:
I think I'll do it in a new heading. The reasons I kept them here are:
1. It's not a small effort going through each plugin and understand which options it uses, how and if the defaults have changed
2. Avoid having broken backlinks to this page. We can't fix those using redirects as the specific options are referenced  using `#`, so we'll need to do some client side routing to fix those.